js/viewComments.js

```diff
@@ -0,0 +1,50 @@
+const allCommentsCount = document.querySelector('.comments-count');
+const commentsView = document.querySelector('.comments-view');
+const commentsLoaderButton = document.querySelector('.social__comments-loader');
+
+const hideComments = (comment) => {
+  comment.classList.add('hidden');
+};
+
+const showComments = (comment) => {
+  comment.classList.remove('hidden');
+};
+
+const viewComments = () => {
+  const comments = document.querySelectorAll('.social__comment');
+  allCommentsCount.textContent = comments.length;
+  if (comments.length <= 5) {
+    commentsView.textContent = comments.length;
+    commentsLoaderButton.classList.add('hidden');
+  } else {
+    commentsView.textContent = 5;
+    for (let i = 0; i < comments.length; i++) {
+      hideComments(comments[i]);
+    }
+    const commentsList = document.querySelector('.social__comments');
+    const hiddenComments = commentsList.querySelectorAll('.hidden');
+    for (let i = 0; i < 5; i++) {
+      showComments(hiddenComments[i]);
+    }
+  }
+};
+
+const onClickShowYet = () => {
+  const commentsList = document.querySelector('.social__comments');
+  const showCommentsFn = () => document.querySelectorAll('.social__comment');
```

Comment:
лучше поменяй название на 
```suggestion
const onClickShowMore = () => {
  const commentsList = document.querySelector('.social__comments');
  const showCommentsFn = () => document.querySelectorAll('.social__comment');
```

а то yet это не совсем то )

Reply:
hiddenCommentsFn и showCommentsFn тоже названия не очень